scripts/docs/snippetadapter.js

```diff
@@ -80,12 +80,13 @@ module.exports = function snippetAdapter( snippets, options, umbertoHelpers ) {
 		snippets.add( snippetData );
 	}
 
-	// Remove snippets that do not match to patterns specified in `options.whitelistedSnippets`.
-	if ( options.whitelistedSnippets ) {
-		filterWhitelistedSnippets( snippets, options.whitelistedSnippets );
+	// Remove snippets that do not match to patterns specified in `options.allowedSnippets`.
+	if ( options.allowedSnippets ) {
+		filterAllowedSnippets( snippets, options.allowedSnippets );
 	}
 
-	console.log( `Building ${ snippets.size } snippets...` );
+	console.log( `Found ${ snippets.size } "{@snippet ...}" tags...` );
+	console.log( `Building ${ countUniqueSnippets( snippets ) } snippets...` );
```

Comment:
The first log is about how many `{@snippet ...}` tags Umberto found. The second one is about how many snippets will be built. The same snippet can be used on many pages but it will be built once.